src/dotnet/commands/dotnet-add/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectJsonMigration;
+using NuGet.Frameworks;
+using Microsoft.DotNet.Tools.Add.ProjectToProjectReference;
+
+namespace Microsoft.DotNet.Tools.Add
+{
+    public class AddCommand
+    {
+        public const string CommandName = "dotnet-add";
+        public const string UsageText = @"Usage: dotnet add [command] [arguments]
+
+Arguments:
```

Comment:
Now after the design changes - first two we could probably live with but third...:
1. Help text is broken - it is counter intuitive
2. no concept of required positional arg
3. we deal with fuzzy positional arg where arg can be either object or the command - it is hard to make a parser which supports that (perhpas: (arg) =>SomeLogic(arg) ? Parser.AcceptArg : Parser.RejectArg)